.env.template

```diff
@@ -5,9 +5,6 @@ FLASK_APP=todo_app/app
 # Turn on debug mode (which enables reloading on code changes and the interactive debugger: https://flask.palletsprojects.com/en/2.3.x/config/#DEBUG)
 FLASK_DEBUG=true
 
-# Required to use Flask's session feature (for storing todos in a browser cookie in the first exercise): https://flask.palletsprojects.com/en/2.3.x/config/#SECRET_KEY
```

Comment:
Nice spot that we can remove this! Note that depending on which exercise you pick in M11 (where there's a choice) then you _may_ want to bring this back (most people choose a different one), but it's much cleaner to remove now regardless 😄 

(Just calling this out because most people don't spot & remove this, therefore the instructions won't necessarily call it out!)